src/InteractiveWindow/Editor/InteractiveWindow.UIThreadOnly.cs

```diff
@@ -39,6 +39,16 @@ private sealed partial class UIThreadOnly : IDisposable
 
             private readonly IInteractiveWindowEditorFactoryService _factory;
 
+            private readonly ITextBufferUndoManagerProvider _textBufferUndoManagerProvider;
+            private ITextBufferUndoManager UndoManager { set; get; }
+            public ITextUndoHistory UndoHistory
```

Comment:
There's a loosely-followd convention to suffix it with _TestOnly so it's clear it's only for tests.